deps/libgit2.version

```diff
@@ -1,2 +1,2 @@
-LIBGIT2_BRANCH=v0.25.1
-LIBGIT2_SHA1=2fcb8705e584ca61f6c4657525c9d2713f6a39d2
+LIBGIT2_BRANCH=v0.26.0
+LIBGIT2_SHA1=15e119375018fba121cf58e02a9f17fe22df0df8
```

Comment:
if you build a new dependency version and the checksum isn't already present, it'll create them the first time - we add them to git so they get checked for everyone else